website1/src/components/Navbar/Navbar.css

```diff
@@ -0,0 +1,145 @@
+.NavbarItems {
```

Comment:
great navbar styling all around. I loved how you made this fit the aesthetics of your site